### PR TITLE
Fix duplicate screenshot, Add init from live version

### PR DESCRIPTION
**Pull request by arsonik**

### CODE REVIEW

**Comment by arsonik** [comment]
One last thing regarding jpEg extension they are not handled by spaceship upload utilities
https://github.com/fastlane/spaceship/blob/master/lib/spaceship/du/utilities.rb#L12

File: lib/deliver/upload_screenshots.rb [hunk 1]
```diff
@@ -49,16 +49,13 @@ def collect_screenshots(options)
...
 
-        files = Dir.glob(File.join(lng_folder, '*.{png,jpg}'))
+        files = Dir.glob(File.join(lng_folder, '*.{png,jpg,jpeg}'))
         next if files.count == 0
 
-        prefer_framed = Dir.glob(File.join(lng_folder, '*_framed.{png,jpg}')).count > 0
+        prefer_framed = Dir.glob(File.join(lng_folder, '*_framed.{png,jpg,jpeg}')).count > 0
 
         files.each do |path|
-          if prefer_framed && !path.include?("_framed.{png,PNG,jpg,JPG}") && !path.downcase.include?("watch")
```
Comment (KrauseFx) [inline lib/deliver/upload_screenshots.rb]: Why did you remove the watch check?

Comment (arsonik) [inline lib/deliver/upload_screenshots.rb]: Sorry didnt mean to do that
